internal/measurement/measurement.go

```diff
@@ -172,69 +176,55 @@ func Percentage(value, total int64) string {
 	}
 }
 
+// unit includes a list of names representing a specific unit and a factor
+// which one can multiple a value in the specified unit by to get the value
+// in terms of the base unit.
+type unit struct {
+	preferredName string
+	names         []string
+	factor        float64
+}
+
+var memoryUnits = []unit{
+	{"B", []string{"b", "byte"}, 1},
+	{"kB", []string{"kb", "kbyte", "kilobyte"}, 1024},
```

Comment:
FWIW / nit: I like expressing IEC constants as shifts of 2: 1 << 10, 1 << 20, 1 << 30, 1 << 40, 1 << 50.

Reply:
Done.